src/types/base/Analytic.ts

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable */
+
```

Comment:
I didn't really understand the way you divided all your models in different folders, so for now I put everything in 1 file that should create the whole request object.

Reply:
Since analytics pertains to the user, it would be consistent to add the analytics request config the user request config collection defined at `src/requests/User.ts`.

Reply:
So the models defined at `type/base` are the models that represent some concrete entities like a single `Tweet` or a single `User`, etc. This has been done for type-safety purposes. Generally, the entities with the field `__typename` are classified as base entities and usually have the same model name as the value defined by the said field.

In this case, I think the IAnalyticResult will be a base entity, so it would be better to make a copy of IAnalyticResult at `type/base/Analytic` and keep the name of the interface the same as the one defined by the field `__typename`,

Reply:
For generating the raw response types, I use [this](https://transform.tools/json-to-typescript) tool and usually keep it as is. Then, I copy the base types into a separate file at `types/base` and change interface names as required.

Reply:
> In this case, I think the IAnalyticResult will be a base entity, so it would be better to make a copy of IAnalyticResult at type/base/Analytic and keep the name of the interface the same as the one defined by the field __typename,

The `__typename` of the analytic request is `User`. Are you implying I need to create the file `type/base/Analytic.ts` with a interface called `IUser`? It is possible but you might run into issues if you ever need the `IUser` object from `User.ts` and the `IUser` object from `Analytic.ts`

Reply:
In that case, let's stick with `IAnalyticResult`, cause that's what makes sense.